src/Common/Core/Test/Fakes/Shell/TestServiceManager.cs

```diff
@@ -2,48 +2,34 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
-using System.Collections.Generic;
+using System.ComponentModel.Composition;
 using System.ComponentModel.Composition.Hosting;
-using System.Linq;
 using Microsoft.Common.Core.Services;
+using Microsoft.UnitTests.Core.Mef;
 
 namespace Microsoft.Common.Core.Test.Fakes.Shell {
-    public class TestServiceManager : IServiceManager {
-        private readonly ExportProvider _compositionContainer;
-        private readonly IServiceManager _services;
+    public class TestServiceManager: ServiceManager {
```

Comment:
I haven't derived `TestServiceManager` from `ServiceManager` so that it can wrap another existing `IServiceManager` without transferring all service instances. This is useful in fixtures when one provides services for the other